index.js

```diff
@@ -110,6 +110,10 @@ var isMultipleOf = function(name, multipleOf) {
 }
 
 var compile = function(schema, cache, root, reporter, opts) {
+  if (Buffer.isBuffer(schema) || schema === null || (typeof schema !== 'string' && typeof schema !== 'object')) {
```

Comment:
No reason to be specific and black-list Buffer.
Instead, `schema === true || schema === false || schema && Object.getPrototypeOf(schema) === Object.prototype` could be whitelisted, here, [like in @exodus/schemasafe](https://github.com/ExodusMovement/schemasafe/blob/de79f893838f1f66f732e5d4850a065c8c6b90b2/src/index.js#L273).

Reply:
Also that resolves the `Buffer` concern.